jpo-geojsonconverter/src/main/java/us/dot/its/jpo/geojsonconverter/serialization/deserializers/OdeSpatDataJsonDeserializer.java

```diff
@@ -23,7 +24,7 @@ public class OdeSpatDataJsonDeserializer implements Deserializer<OdeSpatData> {
 
     private static Logger logger = LoggerFactory.getLogger(OdeSpatDataJsonDeserializer.class);
 
-    protected final ObjectMapper mapper = new ObjectMapper();
+    protected final ObjectMapper mapper = DateJsonMapper.getInstance();
```

Comment:
The custom ObjectMapper seems to cause a problem deserializing the incoming OdeSpatData, and is not necessary here.